<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="2" total="3" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-04-23T18:47:53 CEST" name="Suite" finished-at="2022-04-23T18:48:01 CEST" duration-ms="7955">
    <groups>
    </groups>
    <test started-at="2022-04-23T18:47:53 CEST" name="Test" finished-at="2022-04-23T18:48:01 CEST" duration-ms="7955">
      <class name="uiTests.ErrorTests_motorcycle">
        <test-method is-config="true" signature="setupMethod()[pri:0, instance:uiTests.ErrorTests_motorcycle@1d119efb]" started-at="2022-04-23T18:47:53 CEST" name="setupMethod" finished-at="2022-04-23T18:47:59 CEST" duration-ms="5945" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupMethod -->
        <test-method signature="errorMessage_PayLoadField1()[pri:0, instance:uiTests.ErrorTests_motorcycle@1d119efb]" started-at="2022-04-23T18:47:59 CEST" name="errorMessage_PayLoadField1" finished-at="2022-04-23T18:48:00 CEST" duration-ms="1016" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error message is not as expected on cylinder capacity field. expected [Must be a number between 1 and 3000] but found [Must be a number between 1 and 2000]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error message is not as expected on cylinder capacity field. expected [Must be a number between 1 and 3000] but found [Must be a number between 1 and 2000]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at uiTests.ErrorTests_motorcycle.errorMessage_PayLoadField1(ErrorTests_motorcycle.java:31)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- errorMessage_PayLoadField1 -->
        <test-method is-config="true" signature="cleanUp()[pri:0, instance:uiTests.ErrorTests_motorcycle@1d119efb]" started-at="2022-04-23T18:48:00 CEST" name="cleanUp" finished-at="2022-04-23T18:48:01 CEST" duration-ms="444" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- cleanUp -->
      </class> <!-- uiTests.ErrorTests_motorcycle -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
